modules/vaos/app/services/vaos/appointment_requests_service.rb

```diff
@@ -31,6 +31,28 @@ def get_requests(start_date = nil, end_date = nil)
       end
     end
 
+    def post_request(request_object_body)
+      with_monitoring do
+        params = VAOS::AppointmentRequestForm.new(user, request_object_body).params
+        response = perform(:post, url, params, headers(user))
+
+        {
+          data: OpenStruct.new(response.body)
```

Comment:
Is this a conscious choice to not use a `VAOS::PostResponseObject` or something similar to other clients?

Reply:
Yes, it's kind of a bizarre that the Netflix serialize we're using only accepts an object and not JSON. We don't really have a need for a model. This is just a hack to get the much faster Netflix serializers accept our json.